src/routes/index.ts

```diff
@@ -533,6 +554,22 @@ router.route(config.UPDATE_MANAGING_OFFICER_URL + config.ID)
   .post(...validator.updateManagingOfficerIndividual, checkValidations, updateManagingOfficerIndividual.update);
 router.get(config.UPDATE_MANAGING_OFFICER_URL + config.REMOVE + config.ID, authentication, navigation.hasUpdatePresenter, updateManagingOfficerIndividual.remove);
 
+router.route(config.UPDATE_REVIEW_INDIVIDUAL_MANAGING_OFFICER_URL)
```

Comment:
I don't think you need this either, looks defined above. Probably a merge issue :)

Reply:
Removed